src/commands/profiles/add.ts

```diff
@@ -0,0 +1,61 @@
+import { runCommand } from '../../util/command';
+import { outputFormat } from './util';
+const open = require('open')
+
+const jobTemplateKinds = [
+	'folder_mapping',
+	'personal_drive',
+	'transfer',
+]
+
+function isJobTemplateKindValid(jobTemplateKind) {
+	if (jobTemplateKinds.indexOf(jobTemplateKind) > -1) {
+		return true
+	}
+	return false;
+}
+
+function getJobTemplatesJson(argv, output) {
```

Comment:
Sorry, last thing... `output` is no longer used here.

Reply:
will do, thanks!